Contribute/index.md

```diff
@@ -13,8 +13,6 @@ ms.custom: external-contributor-guide
 
 Welcome to the Microsoft Learn documentation contributor guide!
 
-Sharing your expertise with others on Microsoft Learn helps everyone achieve more. Use the information in this guide to publish a new article to Microsoft Learn or make updates to an existing published article.
-
 Sharing your expertise with others on Microsoft Learn helps everyone achieve more. Use the information in this guide to publish a new article to learn.microsoft.com or make updates to an article that is published already.
```

Comment:
@sazimi Thanks for opening this PR! You're correct that there's a duplicate paragraph here. However, can you edit your PR so that the other paragraph is kept? "Microsoft Learn" is the preferred term here, rather than the URL of learn.microsoft.com. Thank you!

Reply:
Done! ✅